build.gradle

```diff
@@ -11,6 +11,7 @@ buildscript {
     repositories {
         google()
         mavenCentral()
+        jcenter()
```

Comment:
`JCenter` was deprecated this year, so you need to find the color picker lib on some other repository.

Reply:
If you are married to this one, you can use jitpack
`implementation 'com.github.kristiyanP:colorpicker:v1.1.10'`